gson/src/main/java/com/google/gson/JsonArray.java

```diff
@@ -99,6 +99,61 @@ public boolean add(JsonElement element) {
     return elements.add(element);
   }
 
+  /**
+   * Adds the specified boolean to self in the specified position.
+   * All the element on the array will be moved to the right (increment index by one)
+   *
+   * @param index the index where the specified parameter will be added.
```

Comment:
"Parameter" usually describes the named variable in the method signature, e.g. `Bool bool`. It looks like here you are rather writing about an "argument", i.e. a specific value for a parameter. Though using "element" would work as well and might be more specific:
> the index where the specified element will be added.